portfolio-form-submission.js

```diff
@@ -29,6 +29,7 @@ const submitFunc = (event) => {
   let invalidPasswordAlert = document.getElementById("alert-password");
 
   let successMsg = document.getElementById("success-msg");
+  let errorMsg = document.getElementById("error-msg");
```

Comment:
A better impementation of this scenario would be to create one message div instead of creating success-msg and error-msg separately, and handle the content of that div from this function.